client/src/modules/accounts/modals/import.js

```diff
@@ -0,0 +1,68 @@
+angular.module('bhima.controllers')
+  .controller('ImportAccountsController', ImportAccountsController);
+
+ImportAccountsController.$inject = [
+  '$uibModalInstance', 'AccountService', 'Upload', 'NotifyService', '$state',
+];
+
+function ImportAccountsController(Instance, Accounts, Upload, Notify, $state) {
+  const vm = this;
+
+  const IMPORT_DEFAULT_OHADA_ACCOUNTS = 1;
+
+  vm.downloadTemplate = Accounts.downloadAccountsTemplate;
+  vm.cancel = Instance.close;
+
+  vm.select = (file) => {
+    vm.noSelectedFile = !file;
+  };
+
+  vm.submit = () => {
```

Comment:
The `bhLoadingButton` will only show a loading indicator if the `submit()` method returns a promise.  I propose you add `return uploadFile(vm.file)` to this method, and then add `return Upload.upload()` to the `uploadFile()` method to get a nice loading indicator.

Reply:
Also, should this submit method check for `form.$invalid`?